code/modules/jobs/job_types/!crashsite/syndicate_engineer.dm

```diff
@@ -0,0 +1,37 @@
+/datum/job/syndicate_engineer
+	title = "Syndicate Engineer"
+	department_head = list("Requisitions Chief")
```

Comment:
They are apart of the logistics department, and as such listen to the LO, not the RC